enos/modules/run_workloads/templates/vault-config.json.tpl

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
```

Comment:
If we have this header, we end up with invalid JSON when we render the template.